OracleUnifiedDirectory/README.md

```diff
@@ -0,0 +1,134 @@
+Oracle Unified Directory (OUD) on Docker
+========================================
+
+## Contents
+
+1. [Introduction](#introduction)
+2. [Prerequisites for OUD on Docker](#prerequisites-for-oud-on-docker)
+3. [Prerequisites for OUD on Kubernetes](#prerequisites-for-oud-on-kubernetes)
+4. [Installing the OUD Docker Image](#installing-the-oud-docker-image)
+5. [Running the OUD Docker Image in a Docker Container](#running-the-oud-docker-image-in-a-docker-container)
+6. [OUD Docker Container Configuration](#oud-docker-container-configuration)
+7. [OUD Kubernetes Configuration](#oud-kubernetes-configuration)
+
+## Introduction
+
+Oracle Unified Directory provides comprehensive Directory Solution for robust Identity Management.
+
+Oracle Unified Directory is an all-in-one directory solution with storage, proxy, synchronization and virtualization capabilities. While unifying the approach, it provides all the services required for high-performance Enterprise and carrier-grade environments. Oracle Unified Directory ensures scalability to billions of entries, ease of installation, elastic deployments, enterprise manageability and effective monitoring.
+
+This project offers Dockerfiles and scripts to build and configure an Oracle Unified Directory image based on 12cPS4 (12.2.1.4.0) release. Use this Docker Image to facilitate installation, configuration, and environment setup for DevOps users. 
+
+This Image refers to binaries for OUD Release 12.2.1.4.0 and it has the capability to create different types of OUD Instances (Directory Service, Proxy, Replication) on containers targeted for development and testing.
+
+***Image***: oracle/oud:12.2.1.4.0
+
+## Prerequisites for OUD on Docker
+
+The following prerequisites are applicable when using the OUD Docker image on Docker only:
```

Comment:
Instead of "Docker", refer to a "container image" and in standalone mode. The only pre-requisite is an OCI runtime spec compliant container engine, e.g. Docker CE, Moby or Podman.

Reply:
Removed pre-requisites.